pkg/kubelet/server/server.go

```diff
@@ -134,6 +134,8 @@ func ListenAndServeKubeletServer(
 	}
 	if tlsOptions != nil {
 		s.TLSConfig = tlsOptions.Config
+		// Passing empty strings as the cert and key files indicates to call
```

Comment:
nit: grammar: `indicates to call`, I'm not sure what that means. `indicates a need to call`?